cardano-tracer/src/Cardano/Tracer/Handlers/Metrics/Prometheus.hs

```diff
@@ -143,3 +148,14 @@ getMetricsFromNode nodeId acceptedMetrics =
     . T.replace " " "_"
     . T.replace "-" "_"
     . T.replace "." "_"
+
+  metricsCompability :: MetricsList -> MetricsList
+  metricsCompability metricsList =
+      case metricsComp (teConfig tracerEnv) of
+        Nothing -> metricsList
+        Just mmap -> foldl  (\ accu p'@(mn,mv) -> case M.lookup mn mmap of
+                                                Nothing -> p' : accu
+                                                Just rep -> p' : (rep,mv) : accu)
+                            []
+                            metricsList
```

Comment:
Has `metricsList` to be ordered somehow (e.g. identical `mv`s must be subsequent)?
I was wondering about the `fold`...

Reply:
Only one mapping per entry is supported, and then order doesn't matter.